configparser.y

```diff
@@ -776,3 +888,81 @@ append_acl(struct acl_options **list, struct acl_options *acl)
 	}
 }
 
+static int
+parse_boolean(const char *str, int *bln)
+{
+	if(strcmp(str, "yes") == 0) {
+		*bln = 1;
+	} else if(strcmp(str, "no") == 0) {
+		*bln = 0;
+	} else {
+		return 0;
+	}
+
+	return 1;
+}
+
+static int
+parse_number(const char *str, long long *num)
+{
+	/* ensure string consists entirely of digits */
+	size_t pos = 0;
+	while(str[pos] >= '0' && str[pos] <= '9') {
+		pos++;
+	}
+
+	if(pos != 0 && str[pos] == '\0') {
+		int err = errno;
+		errno = 0;
+		*num = strtoll(str, NULL, 10);
+		errno = err;
```

Comment:
The callers of `parse_number` do not use or need the errno. The errno wipe and restore seem unused here, remove it?

Reply:
Done